src/data/valid/MagsAnalysisActivity.yaml

```diff
@@ -0,0 +1,21 @@
+id: nmdc:wfmag-11-547rwq94.1
+ended_at_time: '2021-09-15T10:13:20+00:00'
+execution_resource: NERSC
+git_url: git_url1
+name: MAG analysis for project ABCD
+started_at_time: '2021-08-05T14:48:51+00:00'
+type: nmdc:MagsAnalysisActivity
+was_informed_by: "nmdc:omprc-11-547rwq95"
+has_input:
+  - "nmdc:dobj-11-547rwq96"
+has_output:
+  - "nmdc:dobj-11-547rwq97"
+  - "nmdc:dobj-11-547rwq98"
+mags_list:
+  - bin_name: bins.10 
+    bin_quality: LQ
+    eukaryotic_evaluation:
+      completeness: 14.71
+      contamination: 8.82
+      ncbi_lineage_tax_ids: "1-131567-2759-2611352-33682-191814-2603949"
+      ncbi_lineage: "root,cellular organisms,Eukaryota,Discoba,Euglenozoa,Diplonemea,Diplonemidae" 
```

Comment:
I noticed these two values are enclosed in quotes, whereas the ones in `src/data/invalid/MagsAnalysisActivity-invalid_ncbi_lineage_tax_ids.yaml` are not. I don't know whether that was intentional.

There's also a mixture of single and double quotes in these files. I also don't know whether that was intentional.